src/torchmetrics/__init__.py

```diff
@@ -21,26 +21,19 @@
     SignalNoiseRatio,
 )
 from torchmetrics.classification import (  # noqa: E402
-    AUC,
```

Comment:
do we have replacements for this one?

Reply:
only functional are available now and it has been moved to utilities as it is not really a metric:
https://github.com/Lightning-AI/metrics/blob/fbfe974e44b157819bbe5421494e616bf63c6e6f/src/torchmetrics/utilities/compute.py#L103-L115

Reply:
good, let's do https://github.com/Lightning-AI/metrics/pull/1251#discussion_r992214449